backend/indexer/src/index.ts

```diff
@@ -0,0 +1,26 @@
+import { ponder } from "@/generated";
+import { BigNumber } from "ethers";
+
+ponder.on("EtherspotPaymaster:SponsorSuccessful", async ({ event, context }) => {
+  const { db, network } = context;
+  const { args, block, transaction, log } = event;
+  const time = Number(BigNumber.from(block.timestamp).toString());
+  const timestamp = new Date(time * 1000);
+  try {
+    await db.PaymasterEvent.create({
+      id: transaction.hash + log.logIndex,
+      data: {
+        chainId: network.chainId,
+        sender: args.sender,
+        paymaster: args.paymaster,
+        transactionHash: transaction.hash,
+        timestamp: block.timestamp,
+        month: timestamp.getMonth(),
+        year: timestamp.getFullYear()
+      }
+    })
+  } catch (err) {
+    // caught err
+    console.log(err);
```

Comment:
```suggestion
    console.error(err);
```

Consider a proper logging solution for this in future like `winston`

Reply:
this is on ponder logs we can't do much it still handles the error automatically